lib/interpreters/split_clustering.rb

```diff
@@ -486,10 +499,10 @@ def self.split_hierarchical(service_vrp, nb_clusters, options = {})
         }.flatten
 
         log "Hierarchical Tree (#{end_timer - start_timer}sec): split #{data_items.size} into #{clusters.collect{ |cluster| cluster.data_items.size }.join(' & ')}"
-        cluster_vehicles = assign_vehicle_to_clusters([[]] * vrp.vehicles.size, vrp.vehicles, vrp.points, clusters)
+        # cluster_vehicles = assign_vehicle_to_clusters([[]] * vrp.vehicles.size, vrp.vehicles, vrp.points, clusters)
```

Comment:
You could probably remove it if it is unused

Reply:
We wanted to show how it was supposed to be, but this does not work (entering assign_vehicle_to_clusters would raise an error).